validator/project_validator.go

```diff
@@ -584,6 +585,20 @@ func validateTaskNames(project *model.Project) ValidationErrors {
 	return errs
 }
 
+// validateModuleBranches checks to make sure that branches of modules have a set branch.
+func validateModuleBranches(project *model.Project) ValidationErrors {
+	errs := ValidationErrors{}
+	for _, module := range project.Modules {
+		if module.Branch == "" {
+			errs = append(errs, ValidationError{
+				Level:   Warning,
+				Message: fmt.Sprintf("module '%s' should have a set branch", module.Name),
+			})
+		}
+	}
+	return errs
```

Comment:
Oh, one question: I realize that this is an error not a warning. @ybrill can this affect evergreen yamls in use?

Reply:
I thought it _is_ just a warning, and that this is the function of the Level field.

Reply:
ah yes, you are right. thanks!